### DISCUSSION TITLE
mspider/user-guide/traffic-governance/destination-rules/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1159] [Announcements]
# mspider/user-guide/traffic-governance/destination-rules/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/traffic-governance/destination-rules/

**Comment by liyuerich** [comment]
按屏幕提示选择策略类型，并配置对应的治理策略后 - 有具体对每个治理策略的解释吗？

**Reply by huan555** [reply]
增加：https://gitlab.daocloud.cn/ndx/frontend-engineering/mspider-ui/-/issues/421

**Reply by samzong** [reply]
看下面

**Comment by liyuerich** [comment]
添加服务版本的示例 v2.0和输入框中的提示不一致， v2.0这种格式应该报错的。 是产品的issue还是文档的问题？

**Reply by huan555** [reply]
https://gitlab.daocloud.cn/ndx/frontend-engineering/mspider-ui/-/issues/478

**Reply by samzong** [reply]
不要在这里发内部链接
